### DISCUSSION TITLE
Unhandled Runtime Error Error: async/await is not yet supported in Client Components, only Server Components. This error is often caused by accidentally adding `'use client'` to a module that was originally written for the server.

### DISCUSSION THREAD

**Discussion by saravanansundar823** [vercel/next.js#60426] [Help]
### Summary

how to import gsap in next js 14 from ssr

**Comment by giorgi-gvimradze** [comment]
I resolved it by checking all components that include `use client` if they were exported without `async`. so check all client components if they have that and delete.
